contrib/rackspace/provision-rackspace-cluster.sh

```diff
@@ -5,10 +5,10 @@
 
 set -e
 
-THIS_DIR=$(cd $(dirname $0); pwd) # absolute path
-CONTRIB_DIR=$(dirname $THIS_DIR)
+THIS_DIR="$(cd "$(dirname $0)"; pwd)" # absolute path
```

Comment:
Two sets of quotes should be enough here:
```bash
THIS_DIR=$(cd "$(dirname $0)"; pwd) # absolute path
CONTRIB_DIR=$(dirname "$THIS_DIR")
```
Then quote `"$CONTRIB_DIR" as you did below. I tested this on a Mac in a folder with spaces in the name.